recipes/bluesky-queueserver/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "bluesky-queueserver" %}
+{% set version = "0.0.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bluesky-queueserver-{{ version }}.tar.gz
+  sha256: 07e0a0a065e96c9bc378bda60dd99987d0481951e86ec601152b14ae3a915847
+
+build:
+  number: 0
+  skip: true   # [py<37]
```

Comment:
```suggestion
  skip: true   # [py<37 or win]
```

Reply:
As discussed with @dmgav over DM, this case may be needed, so I'll retract this suggestion.